src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java

```diff
@@ -25,6 +29,8 @@
         implements ColumnVisitor
 {
     private static final JsonParser JSON_PARSER = new JsonParser();
+    private static final List<String> BOOL_TRUE_STRINGS = ImmutableList.of("true", "1", "yes");
```

Comment:
- Keywords will be compared with case-ignored, so we don't need all case variations
  https://github.com/hiroyuki-sato/embulk-parser-jsonpath/pull/13/files#diff-2d0fb975204a675b52cc8ea532734971R90

- Adding more words (e.g. 'on', 'y') is good idea. I'll handle it.